libs/langchain/langchain/chains/query_constructor/base.py

```diff
@@ -1,256 +1,29 @@
-"""LLM Chain for turning a user text query into a structured query."""
-from __future__ import annotations
+from typing import Any, List, Optional, Sequence, Union
```

Comment:
We should keep as much of the provider agnostic code in langchain and not move it, so that providers that move to partner packages are able to re-use it without importing from community

Reply:
@eyurtsev Do you mean to keep the `from __future__ import annotations` line?